action.yml

```diff
@@ -20,3 +26,31 @@ runs:
         file_path="$GITHUB_ACTION_PATH/settings.xml"
         
         echo "path=${file_path}" >> $GITHUB_OUTPUT
+
+    # PDI plugin integration test section
+    - name: Configure vm.max_map_count required by Elasticsearch
+      if: ${{ env.run_plugin_integration_tests }}
+      shell: bash
+      run: |
+        echo "vm.max_map_count=262144" >> /etc/sysctl.conf
+
+    - name: Checkout reusable workflow source repo
+      uses: actions/checkout@v4
+      with:
+        repository: pentaho/actions-common
+        path: actions-common
+        ref: master
```

Comment:
This should be switch to `stable` instead of `master`

Reply:
Updated